packages/app/src/app/pages/common/ErrorBoundary/CodeSadbox/CodeSadbox.tsx

```diff
@@ -1,85 +1,94 @@
+    const [_, setErrorCode] = React.useState<string | null>(null);
 
-  useEffect(() => {
-    codesadboxMounted();
+    useEffect(() => {
+      codesadboxMounted();
 
-    const sentryCode = captureException(error);
-    setErrorCode(sentryCode);
-  }, [codesadboxMounted, error]);
+      const sentryCode = captureException(error);
+      setErrorCode(sentryCode);
+    }, [codesadboxMounted, error]);
 
-  return (
-    <Element
-      style={{ width: '100vw', height: '100vh', background: 'rgb(17,21,24)' }}
-    >
-      <Navigation title="CodeSadbox" />
-      <Container>
-        <Content>
-          <Title>Oh no! Something broke!</Title>
-          <Sadbox scale={3} />
-          <Subtitle>CodeSadbox</Subtitle>
-          <Actions>
-            {isLoggedIn ? (
-              <Button small secondary href={dashboardUrl()}>
-                <ButtonIcon>
-                  <Dashboard />
-                </ButtonIcon>
-                Go to Dashboard
-              </Button>
-            ) : (
-              <Button small secondary href="/">
-                <ButtonIcon>
-                  <GoHome />
-                </ButtonIcon>
-                Back to Home
-              </Button>
-            )}
+    return (
+      <ThemeProvider theme={theme.vsCode}>
+        <Element
+          css={css({
+            width: '100vw',
+            overflow: 'hidden',
+            backgroundColor: 'sideBar.background',
 
-            <Button
-              small
-              target="_blank"
-              rel="noopener"
-              href={buildCrashReport({ error, trace, errorCode })}
-            >
-              <ButtonIcon>
-                <GoIssueOpened />
-              </ButtonIcon>
-              Report Crash
-            </Button>
-          </Actions>
-        </Content>
-      </Container>
-    </Element>
-  );
-};
+            a: {
+              textDecoration: 'none',
+            },
+          })}
+        >
+          <Navigation title="CodeSadbox" />
+          <Container>
+            <Content>
+              <Title>Oh no! Something broke!</Title>
+              <Sadbox scale={3} />
+              <Subtitle>CodeSadbox</Subtitle>
+              <Stack gap={2}>
+                {isLoggedIn ? (
+                  <a href={dashboardUrl()}>
+                    <Button autoWidth variant="secondary">
+                      <ButtonIcon>
+                        <Dashboard />
+                      </ButtonIcon>
+                      Go to Dashboard
+                    </Button>
+                  </a>
+                ) : (
+                  <a href="/">
+                    <Button autoWidth variant="secondary">
+                      <ButtonIcon>
+                        <GoHome />
+                      </ButtonIcon>
+                      Back to Home
+                    </Button>
+                  </a>
+                )}
+                <a href={buildCrashReport({ error, trace })}>
```

Comment:
Can you bring the `errorCode` back here?

Reply:
I have no idea why I deleted it tbh